spec/examples/fail_with_error_with_custom_failure_class.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class FailWithErrorWithCustomFailureClass < ApplicationService
```

Comment:
We could set the classes here directly instead of in a different file, for the same result, I think:

```suggestion
class FailWithErrorWithCustomFailureClass < Actor
  self.argument_error_class = MyCustomArgumentError
  self.failure_class = MyCustomFailure

```

Reply:
I have already renamed `ApplicationService` to `ApplicationServiceWithCustomClasses`. But I still like the idea of getting rid of the `ApplicationService*` class in the examples.